garak/cli.py

```diff
@@ -237,6 +238,11 @@ def main(arguments=[]) -> None:
         action="store_true",
         help="Launch garak in interactive.py mode",
     )
+    parser.add_argument(
+        "--skip_unknown",
+        action="store_true",
+        help="allow skip of unknown probes or detectors",
```

Comment:
Only probes & detectors? I guess harness and generators we always want to just stop - what about buffs?

Reply:
Expanding message to extend to `buffs` as well.